src/java/org/apache/cassandra/db/filter/RowFilter.java

```diff
@@ -68,19 +68,22 @@ public abstract class RowFilter implements Iterable<RowFilter.Expression>
 
     protected final List<Expression> expressions;
 
-    protected RowFilter(List<Expression> expressions)
+    private final boolean isStrict;
```

Comment:
This property being a part of the row filter is one of the big decisions made in this patch. One of the reasons it felt appropriate is that `RowFilter` itself is still broken around CASSANDRA-19007. The property being present here might make it easier for the logic below to adjust. I'm not sure if that belongs in this Jira, but if there is a clear path to fixing it, perhaps I can create another one and fast-follow after I commit this patch?